Resources/config/config.yml

```diff
@@ -2,6 +2,10 @@ eo_airbrake:
   api_key: "%errbit_api_key%"
   host: errors.sumocoders.be
 
+sumo_coders_framework_error:
+  show_messages_for:
+    - \Symfony\Component\HttpKernel\Exception\NotFoundHttpException
```

Comment:
is die trailing slash in het begin nodig? Je zult toch altijd het volledige pad moeten gebruiken?